README.md

```diff
@@ -4167,6 +4167,8 @@ P.S. That is my cat :cat Цезарь
 
 ![Caesar](assets/images/Caesar.jpg)
 
+<<<<<<< HEAD
```

Comment:
You need to remove those lines with "<<<<<<<< HEAD" - it is a part of the conflict resolving message.